webapp/src/serial.tsx

```diff
@@ -75,32 +73,21 @@ export class Editor extends srceditor.Editor {
 
         const data = smsg.data || ""
         const source = smsg.id || "?"
-        let theme = source.split("-")[0] || "black"
```

Comment:
how do you differentiate messages coming from different micro:bits?

Reply:
We don't need to support that now, but I assume the message IDs would be different.